app/api/routes.py

```
@@ -401,4 +405,69 @@ def delete_record_route(
-        raise HTTPException(status_code=500, detail="기록 삭제 중 서버 오류가 발생했습니다.") from e
\ No newline at end of file
+        raise HTTPException(status_code=500, detail="기록 삭제 중 서버 오류가 발생했습니다.") from e
+
+
+@router.get("/api/v1/consult/start",
+     tags=["에이전트 상담"],
+     summary="새로운 복막투석 상담 시작",
+     description="새로운 복막투석 상담 세션을 시작하고 고유한 세션 ID를 발급합니다.",
+     response_model=SessionStartResponse,
+)
+def start_chat_session(current_user: User = Depends(AuthTokenDep)):
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)